syntax = "proto3";
package prpc;
message RpcHeader {
    bytes servic_name = 1;
    bytes method_name = 2;
    uint32 args_length = 3;
}
